### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by anewuser** [uBlockOrigin/uBlock-issues#3262] [Ideas]
-

**Comment by Yuki2718** [comment]
Not something I want to spend my time.

**Reply by gorhill** [reply]
Right, best is that someone motivated creates/maintains such list, then eventually if it's popular, we will consider about adding it to uBO's stock lists.

**Reply by peace2000** [reply]
I haven't had resource issues with Dark Reader. It usually generates themes quite well, though not always. Dark themes provided by pages are of course better as they are custom designed.

I'm not against such a list in uAssets though it would be more meaningful if bigger amount of people ask for it. I'd subscribe to that list if such a list were created.

**Reply by garry-ut99** [reply]
> anewuser: What do you think about creating a uAssets list with rules that enable those themes by default? 

It seems it's a good idea to create such a filter list, technically it's one more kind of annoyances not different than others, on which devs already spend their time, it doesn't seem like one more kind of annoyance to fix would make such a big difference to "not spend time on it", if a need to address some annoyances appear on the scene, it should be addressed like any other annoyoances.

> Yuki2718: Not something I want to spend my time.
> > gorhill: right

Right what? He didn't even provide a single reason why he doesn't want to spent time on it, yet you immediately have agreed and are blind-guessing his reasons, also you didn't wait for other devs opinions, perhaps they will be interested (or perhaps not), by the way - other devs opinions might depend on community opinion.

> peace2000: I'm not against such a list in uAssets. I'd subscribe to that list if such a list were created.

Same here.

> peace2000: though it would be more meaningful if bigger amount of people ask for it.  

Maybe a good idea would be if anewuser would post the proposal on uBlock reddit and let the whole community respond, it might turn out that there are enough users who would subscribe to such filter list.

**Reply by gorhill** [reply]
> Right what?

I think we are all already burden enough. The way I see it is that whoever has an idea for a list should work on it rather than ask current current maintainers to work more.

> He didn't even provide a single reason why he doesn't want to spent time on it

Not spending time is the reason. We are all volunteers here, no one should have to justify why we don't want to spent more time on working on issue.

Bottom line, we need more volunteers, not more workload added to current volunteers.

**Reply by gorhill** [reply]
If you are willing to contribute to it, I don't have any objection to host the list here, If it becomes popular after a while, I will consider adding it to stock lists.

**Reply by garry-ut99** [reply]
> gorhill : I think we are all already burden enough. The way I see it is that whoever has an idea for a list should work on it rather than ask current current maintainers to work more. Bottom line, we need more volunteers, not more workload added to current volunteers.

You think what you think, but he only said he doesn't want to spend time on it, which could be understood differently, for example as he was just not interested in the idea, and there could be plenty of reasons why he isn't interested, by the way, since you only think, but are not sure, maybe better to ask them, and let them speak for themselves, because as seen: Yuki said "no", but peace said "yes".

> gorhill : Not spending time is the reason. We are all volunteers here, no one should have to justify why we don't want to spent more time on working on issue.

So you mean that "if he wanted to, he would have already written the reasons", perhaps, but on the other hand maybe he just didn't want to elaborate on it, but he still wouldn't mind writing back if someone asked, maybe instead of being an advocate of all volunteers or trying to guess the context, to let them decide for themselves whether they want or not to reply to questions from community, people are only asking and discussing, questions and discussion is allowed, and is not a crime, it's what Discussions are for, and since volunteers create filters not only for themselves but for the whole community, it's expected to hear a few words from them, more than "no because no", unless you already decided for them, in such case there is no place for discussion anymore, it seems to be this:
> https://github.com/uBlockOrigin/uBlock-issues/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml
> New filter list requests are not permitted and will be declined and closed.

**Reply by peace2000** [reply]
@gorhill If you add that list to uAssets repo, I can maintain it. Though would `trusted` rules on it be accepted by uBO as it would not be a stock list? That list would probably heavily rely on setting cookies and it's expected that `trusted` rules would be needed.

**Reply by garry-ut99** [reply]
> Though would `trusted` rules on it be accepted by uBO as it would not be a stock list? 

It won't, until an user manually adds a token or url for imported list to [trustedListPrefixes](https://github.com/gorhill/uBlock/wiki/Advanced-settings#trustedListPrefixes)
